soco/core.py

```diff
@@ -1490,6 +1493,59 @@ def get_item_album_art_uri(self, item):
         else:
             return None
 
+    @only_on_master
+    def set_sleep_timer(self, sleep_time_seconds):
+        """Sets the sleep timer.
+
+        Args:
+            sleep_time_seconds (`int`): How long to wait before turning off
+                speaker in seconds. Maximum value of 86399
+
+        Returns:
+            bool: True if succesful, False otherwise
+
+        Raises SoCoException (or a subclass) upon errors, ValueError for
+            incorrect syntax.
+
+        """
+        if sleep_time_seconds and \
```

Comment:
This string manipulation code is ugly and seeing as I'm new to Python, I'm open to any examples you can offer.

Also, as I was preparing this comment, I realized my code doesn't distinguish between 0 and '' for ```sleep_time_seconds```. These values are different, '' will stop the current sleep timer, but '' will cause the speaker to immediately start tapering the sound, a potentially useful feature. I wonder if you can work this into such an example, or if we should have a separate method just for this feature.